Automattic/sensei
# Standardize button styles

### Changes proposed in this Pull Request

* This PR makes some styles be sharable between the button blocks.
* Update the block name and default text to _"Contact Teacher"_
* Use the same alignment behavior to Take Course and Contact Teacher buttons. Notice that using the _"Link"_ style for the Contact Teacher block, the alignments _"Align center"_ or _"Full content width"_ will have the same behavior (I thought to make the clickable area full when it's full alignment, but I think it wouldn't make sense)

### Testing instructions

* Create a course.
* Add the Take Course and Contact Teacher buttons to a course.
* Change the alignments.
* Make sure both have the same behavior and they work properly in the editor and in the frontend.
